src/platform/plugins/shared/dashboard/public/dashboard_api/types.ts

```diff
@@ -209,15 +223,19 @@ export type DashboardApi = CanExpandPanels &
     isManaged: boolean;
     locator?: Pick<LocatorPublic<DashboardLocatorParams>, 'navigate' | 'getRedirectUrl'>;
     panels$: PublishingSubject<DashboardPanelMap>;
+    sections$: PublishingSubject<DashboardSectionMap>;
     runInteractiveSave: () => Promise<SaveDashboardReturn | undefined>;
     runQuickSave: () => Promise<void>;
     scrollToPanel: (panelRef: HTMLDivElement) => void;
     scrollToPanelId$: PublishingSubject<string | undefined>;
     scrollToTop: () => void;
+    scrollToBottom$: Subject<void>;
```

Comment:
Nit: should dashboardApi.scrollToBottom$ really be exposed in dashboardApi? When would an embeddable ever need this? How about moving `scrollToBottom$` to internalApi since it seems like an implementation detail of the dashboard.

Reply:
Good point!! I wanted `scrollToTop` and `scrollToBottom` to be in the same spot, but it'll take a bit more work to move `scrollToTop` to internal API because the controls popover is missing the internal context provider. But ultimately we probably don't need to expose this, so I moved `scrollToBottom` to internal API in cbe36ebf83d81808afc823acb5dab710297dac35. We can move the other stuff to `internalApi` in a follow up :+1: